doorway-ui-components/src/global/text.scss

```diff
@@ -146,7 +146,7 @@ h1.title {
 
 .text__small-normal {
   font-family: var(--text-small-normal-font-family, var(--bloom-font-alt-sans));
-  font-weight: var(--text-small-normal-font-weight, normal);
+  font-weight: var(--text-small-normal-font-weight, 400);
```

Comment:
@lisunshiny Hey there! Just some retrospective feedback here - an easier and preferred way to make this change would have been to override the `text-small-normal-font-weight` variable rather than make a change in the text.scss file itself. Those variables are there exactly to make cases like this easier and prevent local changes :) Let me know if you'd like to touch base!